prime-router/src/main/kotlin/cli/tests/LoadTests.kt

```diff
@@ -102,14 +102,45 @@ class LongLoad : LoadTestSimulator() {
         val afterActionId = getMostRecentActionId()
         val results = mutableListOf<SimulatorResult>()
         var elapsedTime = measureTimeMillis {
-            repeat(10) {
-                results += productionSimulation(environment, options)
-                results += productionSimulation(environment, options)
-                results += runOneSimulation(typicalStracSubmission, environment, options) // strac
-                results += productionSimulation(environment, options)
-                results += productionSimulation(environment, options)
-                results += productionSimulation(environment, options)
+            repeat(2) {
```

Comment:
Because of the 50ms delay, this is a tamed down test compared to the original productionSimulation.